draft-ietf-quic-tls.md

```diff
@@ -1287,6 +1287,13 @@ Similarly, there is no need to increase the number of allowed streams until the
 handshake completes.
 
 
+### Handshake Failures
+
+The `CONNECTION_CLOSE` frame MAY be sent by either endpoint during the handshake
+to signal a fatal error with connection establishment.  A `STREAM` frame
+carrying a TLS alert MAY be included in the same packet.
+
```

Comment:
Maybe we should add a note in the next section, "Denial of Service with Unprotected Packets". Sending "connection close" frames in Server Clear Text or Client Clear Text packets would be another way to deny connection.

Reply:
I'll do that separately: see https://github.com/quicwg/base-drafts/pull/908/files